assets/src/edit-story/components/keyboard/index.js

```diff
@@ -143,12 +160,20 @@ export function useGlobalKeyUpEffect(
   callback,
   deps = undefined
 ) {
-  if (!globalRef.current) {
-    globalRef.current = document;
-  }
+  setGlobalRef();
   useKeyUpEffect(globalRef, keyNameOrSpec, callback, deps);
 }
 
+/**
+ * @param {string|Array|Object} keyNameOrSpec
+ * @param {Array|undefined} deps
+ * @return {boolean} Stateful boolean that tracks whether key is pressed.
+ */
+export function useGlobalIsKeyPressed(keyNameOrSpec, deps = undefined) {
+  const globalRef = getGlobalRef();
```

Comment:
So which form did you decide to use? `setGlobalRef()` or `const globalRef = getGlobalRef()`? I only see `setGlobalRef()` declaration above.

Reply:
My bad, I accidently left this one in. Fixed.